src/pages/Home/components/Map/components/Location/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactComponent as Search } from 'assets/svg/home/search.svg';
+import { ReactComponent as Point } from 'assets/svg/home/point.svg';
+import styles from './Location.module.scss';
+
+export default function Location(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <div className={styles.box}>
+        <div className={styles.box__title}>현재 위치가 올바르지 않은가요?</div>
+        <div className={styles.box__subTitle}>현재 계신 곳의 위치를 아래 검색창을 통해</div>
+        <div className={styles.box__subTitle}>알려주시면 반영하겠습니다.</div>
```

Comment:
간단한 줄바꿈이라면 `<br />`태그를 사용하는건 어떨까요?
line-height을 적절히 활용할 수 있고, 수정도 용이하다고 생각해요
```suggestion
        <div className={styles.box__subTitle}>
          현재 계신 곳의 위치를 아래 검색창을 통해
          <br />
          알려주시면 반영하겠습니다.
        </div>
```

Reply:
이 코드를 사용한다면 각 문장에 추가로 태그 처리를 해줘야 코드를 묶어서 중앙정렬을 해줄수 있어 코드의 길이가 더 늘어나지 않을까요?!

Reply:
`text-align: center` 를 사용하면 내부 태그 처리가 따로 없어도 될 거예요